EIPS/eip-4519.md

```diff
@@ -4,7 +4,7 @@ title: Non-Fungible Tokens Tied to Physical Assets (SmartNFT)
 description: Standard interface for non-fungible tokens representing physical assets that can generate or recover their own accounts and obey users.
 author: Javier Arcenegui (@Hardblock-IMSE-CNM), Rosario Arjona <arjona@imse-cnm.csic.es>, Roberto Román <roman@imse-cnm.csic.es>, Iluminada Baturone <lumi@imse-cnm.csic.es>
```

Comment:
Is it possible to get more authors' GitHub handles in this? They are preferred to emails.